### PR TITLE
Add Component design

### PR DESCRIPTION
**Pull request by Katended**
Th e following have been addressed in this pull request:

Created a new directory - components.
Added Calculator.js file.
Created at least 1 child component

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    </div>
+  </section>
+);
+
+const Calculator = () => (
+  <div>
+    <CalcBody />
+  </div>
+);
+
```
Comment (roshan-bajgain) [inline src/components/Calculator.js:37]: - [ ] According to the project requirement your app should match the following design. Please implement it.
### Your design
his

### Original design
original

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```
Comment (roshan-bajgain) [inline src/index.js]: - [ ] I would suggest you that your App.js(main component) should have the imported Calculator.js component and the `App` should be used here in index.js as it is generated by React.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Katended, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Please make sure to delete all the boilerplate from Create React App (text, images, styles) that is on `public` folder at top and also in the `src` folder
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (TadesseDev) [inline src/index.js]: - [ ] Dear @Katended, You did well making the most of the changes but, as suggested by the previous reviewer, your App.js(main component) should have the imported Calculator.js component and the App should be used here in index.js as it is generated by React.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Katended 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Good job making most of the changes requested by the previous reviewer. 🙏


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Katended) [reply src/index.js]: @TadesseDev I am not sure but I seem to find this requested changes already existing in the codebase. I have made a [commit at](https://github.com/Katended/math-magicians/commit/ef684d462ad4514eb938b72d64c3c915e1f8bac4) . And it seems to suggest so. Please let me know.

Comment (TadesseDev) [reply src/index.js]: - dear @Katended, It seems like you pushed commits while I was reviewing (I can also see you are still pushing commits), which are not part of my reviews. Usually, when you submit a review you should have added all the changes. Please verify all the changes are in place and ask for a re-review.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Katended , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**